src/stdatamodels/jwst/transforms/models.py

```diff
@@ -1989,6 +1991,164 @@ def _evaluate_transform_guess_form(model, x=0, y=0, t=0):
     raise TypeError(f"Expected a model or list of models, but got {type(model)}. ")
 
 
+class MIRIWFSSBackwardDispersion(_BackwardGrismDispersionBase):
+    """Calculate the dispersion extent of MIRI WFSS pixels."""
+
+    def __init__(self, orders, lmodels=None, xmodels=None, ymodels=None, name=None):
+        """
+        Initialize the model.
+
+        Parameters
+        ----------
+        orders : list
+            The list of orders which are available to the model. For MIRIWFSS orders = 1.
+        lmodels : list[1D models]
+            The list inverse dispersion 1D polynomial models. Use to determine the trace parameter.
+        xmodels : list[1D models]
+            The list of 1D polynomial models in x. It depends on the trace parameter
+        ymodels : list[2D models]
```

Comment:
```suggestion
        ymodels : list[2D models]
```
Isn't this a list of lists? If so would you update this docstring?

Reply:
Even with 1 order isn't it a list of lists?
https://github.com/spacetelescope/stdatamodels/blob/1577c46042d600f581117b19710f460cbebdd77c/src/stdatamodels/jwst/transforms/models.py#L2063-L2065
first indexes into to `self.ymodels` using the `iorder` to get `ymodel` and then indexes using 0, 1, 2 as in your example.

Reply:
Sounds great!